draft-ietf-tls-subcerts.md

```diff
@@ -291,21 +299,25 @@ Client            Front-End            Back-End
 
 These two mechanisms can be complementary.  A server could use credentials for
 clients that support them, while using [KEYLESS] to support legacy clients.
+The private key for a delegated credential can be used in place of a certificate
+private key, so it is important that the Front-End and Back-End are parties that
+have a trusted relationship in the diagram above.
 
-It is possible to address the short-lived certificate concerns above by
-automating certificate issuance, e.g., with Automated Certificate Managmeent
-Encvironment (ACME) {{?RFC8555}}.  In addition to requiring frequent
-operationally-critical interactions with an external party, this makes
-the server operator dependent on the CA's willingness to issue certificates
-with sufficiently short lifetimes.  It also fails to address the issues with
-algorithm support.  Nonetheless, existing automated issuance APIs like ACME
-may be useful for provisioning credentials within an operator network.
+Use of short-lived certificates with automated certificate issuance,
+e.g., with Automated Certificate Managment Environment (ACME) {{?RFC8555}},
+reduces the risk of key compromise, but has several limitations.
+Specifically, it introduces operationally-critical interactions with an
+external party and forces a server operator dependency on the CA's
+willingness to issue certificates with sufficiently short lifetimes.  It also
```

Comment:
```suggestion
external party.  It also
```

I realize that you have to find a CA that is willing to issue short-lived certificates, but that is obvious enough that you don't need to say it.